app/assets/stylesheets/views/_topics.scss

```diff
@@ -123,7 +123,7 @@
   .browse-container {
     background-color: $page-colour;
     min-height: 500px;
-    padding: 0 30px 45px 30px;
+    padding: 0 0 45px 0;
```

Comment:
I'm not sure I agree with it being an anti-pattern.  Even post you've linked to it mentions things being less DRY as a caveat of using longhand.  I think it is easier to look at first glance if it's shorter, and if this were a function, four arguments wouldn't necessarily be that many.